src/applications/claims-status/components/claim-status-tab/WhatYouNeedToDo.jsx

```diff
@@ -5,8 +5,16 @@ import { getFilesNeeded } from '../../utils/helpers';
 import FilesNeeded from '../claim-files-tab/FilesNeeded';
 
 function WhatYouNeedToDo({ claim }) {
-  const { trackedItems } = claim.attributes;
-  const filesNeeded = getFilesNeeded(trackedItems);
+  const { trackedItems, evidenceWaiverSubmitted5103 } = claim.attributes;
+  const filesNeeded = trackedItems
+    ? getFilesNeeded(trackedItems).filter(
+        i =>
+          !(
+            evidenceWaiverSubmitted5103 &&
```

Comment:
I'd love to see a comment here explaining why we are filtering this out. I think I see what's going on here, but a comment would help future generations. 
If I am correct, then it could be something like: `// If the user has indicated they will not be submitting more evidence by adding a 5103 waiver, do not show them they need to respond to an automated 5103 request`

Reply:
sure I'll add a comment :)

Reply:
@shaunburdick comment has been added :)